sys/tools/syscalls/core/scarg.lua

```diff
@@ -0,0 +1,153 @@
+--
+-- SPDX-License-Identifier: BSD-2-Clause
+--
+-- Copyright (c) 2024 Tyler Baxter <agge@FreeBSD.org>
+-- Copyright (c) 2023 Warner Losh <imp@bsdimp.com>
+-- Copyright (c) 2019 Kyle Evans <kevans@FreeBSD.org>
```

Comment:
To pick a file where the original code is mostly mine or derived from mine, there should likely be an SRI copyright line in the general copyright blocks:
```
-- Copyright (c) 2021-2024 SRI International
```

Reply:
I've added this copyright to scarg.lua and config.lua. There's a reasonable amount of your work in config.lua. Let me know any other locations!